Update CHANGELOG for 1.0.0

czosel opened the pull request:
I'm not completely sure how the update path to 1.0.0 looks. I suppose `changeset` and `changeset-validations` will be removed from `node_modules`. Will the blueprint be automatically installed? If not, we'd probably have to suggest installing the two dependencies manually if they're required?

bendemboski commented:
@czosel that's correct, the blueprint only runs when the addon is initially installed. Users can manually run the blueprint by running `ember generate ember-validated-form`, but I don't think this is really the best path in this case. Since the only thing it does is install those two addons, I think it's better to just tell the users to install them themselves if they need to so there isn't any mystery around what's happening.

So I think all we really need to say (somewhere) is something like:

> The 1.0.0 release removes `ember-changeset` and `ember-changeset-validations` as dependencies because this addon is built to make them very easy to use, but doesn't require them. So if you are using either or both of them in your application code but do not already have them in your `package.json`, you should run `ember install ember-changeset` and/or `ember install ember-changeset-validations`.

And then I guess we should also say something about switching from `on-submit=task` to `on-submit=(perform task)`

czosel commented:
@bendemboski Alright! I agree that just recommending to install the two addons manually makes most sense. I updated the description in the changelog and also added a reference to the migration docs of the `on-submit` change. What do you think?

bendemboski commented on CHANGELOG.md:
````diff
@@ -4,6 +4,21 @@ All notable changes to this project will be documented in this file.
+
+```bash
+ember install ember-changeset
+ember install ember-changeset
````
I think you meant this second one to be `ember-changeset-validations`, right?

czosel replied:
Oops, thanks 👍
